src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -1232,7 +1251,9 @@ TransformSubqueryNode(MultiTable *subqueryNode, List *tableNodeList)
  * worker nodes' results.
  */
 static MultiExtendedOp *
-MasterExtendedOpNode(MultiExtendedOp *originalOpNode)
+MasterExtendedOpNode(MultiExtendedOp *originalOpNode, bool
```

Comment:
`bool` is missing the variable on this line. Could you check for other examples and also fix them?